Space.js

```diff
@@ -1,63 +1,59 @@
 module.exports = function Space(key, reporters, errback) {
-
   function forEachReporter(func) {
-    reporters.forEach(function(reporter) {
+    reporters.forEach(reporter => {
       try {
         func(reporter);
-      }
-      catch (e) {
-        var errMsg = e && e.message ? e.message : 'Error in reporter';
+      } catch (e) {
+        const errMsg = e && e.message ? e.message : 'Error in reporter';
+        // eslint-disable-next-line no-console
```

Comment:
if you want console, just allow it in eslint, no ?

Reply:
I want it only in specific places - I plan to create an internal logger to abstract this - allowing to be plugged in externally to a standard log on the application side